src/main/webapp/scripts/constant-script.js

```diff
@@ -20,7 +20,7 @@ const GOOGLE_API_KEY = "AIzaSyDlLtx69Y4-65_dCK67ZX3lzKTYpyc5CWI";
 document.addEventListener("DOMContentLoaded", () => {
   const sidenavElems = document.querySelectorAll(".sidenav");
   const sideNavInstances = M.Sidenav.init(sidenavElems, undefined);
-  const modalElems = document.querySelectorAll('.modal');
+  const modalElems = document.querySelectorAll(".modal");
```

Comment:
nit: according to Google style, single quotes are preferred over double quotes for string literals

Reply:
We should fix this in a later PR. I'm using the Prettier VS Code extension, and it mostly adheres to great JS style, except for the string literals. I'll change my config file for Prettier and we can make a formatting fix PR afterwards.